Lombiq.NodeJs.Extensions/Docs/JavaScript.md

```diff
@@ -81,25 +85,7 @@ If a certain rule's violation is incorrect in a given location, or you want to s
 
 Visual Studio supports ESLint out of the box. You can enable it by ticking the checkbox "Enable ESLint" under _Tools → Options → Text Editor → JavaScript/TypeScript → Linting → General_. To use ESLint in Visual Studio Code, you can use e.g. Microsoft's official [ESLint plugin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
 
-In order for Visual Studio to use the ESLint configuration provided by Node.js Extensions instead of its own, it needs to be able to access the necessary ESLint plugins. Here's how to set this up:
-
-#### At the project level
-
-1. `<NodeJsExtensionsGlobalESLintConfigurationDirectory>` is empty or not set.
-2. If the consuming project does not contain a _package.json_ file yet, Node.js Extensions will create it.
-3. Building your project will install the necessary dependencies.
-
-#### At the solution level
-
-1. `<NodeJsExtensionsGlobalESLintConfigurationDirectory>` is set to a valid directory path.
-2. If a _package.json_ file does not exist at that path yet, Node.js Extensions will create it (but only when using Node.js Extensions from a submodule).
-3. Building your solution will install the necessary dependencies.
-
-Afterwards, Visual Studio will show ESLint warnings already during development, using the same configuration that will be used during the build.
-
-#### Maintenance
-
-In order to keep the ESLint rules and configuration in-sync between Visual Studio and Node.js Extensions, make sure all of the package dependencies in your _package.json_ starting with `eslint` always use the same versions as those in Node.js Extensions' _package.json_, especially after updating the latter.
```

Comment:
Is this not needed anymore?

Reply:
No. We update those automatically from NE.